library/cephadm_bootstrap.py

```diff
@@ -149,37 +156,77 @@
 RETURN = '''#  '''
 
 
-def main() -> None:
+def run_module() -> None:
+
+    cephadm_params = dict(
+        docker=dict(type='bool', required=False, default=False),
+        image=dict(type='str', required=False),
+        pull=dict(type='bool', required=False, default=True),
+    )
+
+    cephadm_bootstrap_params = dict(
+        allow_fqdn_hostname=dict(type=bool, required=False, default=False),
+        allow_mismatched_release=dict(type=bool, required=False),
+        allow_overwrite=dict(type=bool, required=False, default=False),
+        apply_spec=dict(type=str, required=False),
+        cluster_network=dict(type=str, required=False),
+        config=dict(type=str, required=False),
+        dashboard_crt=dict(type=str, required=False),
+        dashboard_key=dict(type=str, required=False),
+        dashboard_password_noupdate=dict(type=bool, required=False),
+        fsid=dict(type=str, required=False),
+        initial_dashboard_password=dict(type=str, required=False),
+        initial_dashboard_user=dict(type=str, required=False),
+        log_to_file=dict(type=bool, required=False),
+        mgr_id=dict(type=str, required=False),
+        mon_addrv=dict(type=str, required=False),
+        mon_id=dict(type=str, required=False),
+        mon_ip=dict(type=str, required=True),
+        no_cleanup_on_failure=dict(type=bool, required=False),
+        no_minimize_config=dict(type=bool, required=False),
+        orphan_initial_daemons=dict(type=bool, required=False),
+        output_config=dict(type=str, required=False),
+        output_dir=dict(type=str, required=False),
+        output_keyring=dict(type=str, required=False),
+        output_pub_ssh_key=dict(type=str, required=False),
+        registry_json=dict(type=str, required=False),
+        registry_password=dict(type=str, required=False, no_log=True),
+        registry_url=dict(type=str, required=False),
+        registry_username=dict(type=str, required=False),
+        shared_ceph_folder=dict(type=str, required=False),
+        single_host_defaults=dict(type=bool, required=False),
+        skip_admin_label=dict(type=bool, required=False),
+        skip_dashboard=dict(type=bool, required=False, default=True),
```

Comment:
```suggestion
        skip_dashboard=dict(type=bool, required=False, default=False),
```
I think all the `--skip-*`  parameters should be `default=False`